data/bash-completion/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# Copyright (C) 2006-2020 Istituto Italiano di Tecnologia (IIT)
+# All rights reserved.
+#
+# This software may be modified and distributed under the terms of the
+# BSD-3-Clause license. See the accompanying LICENSE file for details.
+
+yarp_install(FILES yarp
+             DESTINATION ${CMAKE_INSTALL_DATADIR}/bash-completion/completion
```

Comment:
@drdanz 

According to this [post][1], we have:
```sh
$ pkg-config --variable=completionsdir bash-completion
/usr/share/bash-completion/completions
$
$ pkg-config --variable=compatdir bash-completion
/etc/bash_completion.d
```

Therefore:
- it seems that the location where yarp completion gets copied into should end with `completions` and not `completion`
- anyway, that location is not automatically recognized by the system; thus, is there any way to include it?


[1]: https://serverfault.com/a/506707